### ISSUE TITLE
Update Help Documentation example

### ISSUE THREAD

**Issue by mobias17** [firoorg/firo#1330]
Some functions seem to require an update in the `help `documentation for curl exmaples. 

### mintspark

```
mintspark {"address":amount,memo...}


Arguments:
    {
      "address":amount   (numeric or string) The Spark address is the key, the numeric amount (can be string) in FIRO is the value
      ,...
    }

Result:
"txid" (string) The transaction id for the send. Only 1 transaction is created regardless of 
                                    the number of addresses.

Examples:

Send two amounts to two different spark addresses:
> firo-cli mintspark "{\"sr1xtw3yd6v4ghgz873exv2r5nzfwryufxjzzz4xr48gl4jmh7fxml4568xr0nsdd7s4l5as2h50gakzjqrqpm7yrecne8ut8ylxzygj8klttsgm37tna4jk06acl2azph0dq4yxdqqgwa60\":{\"amount\":0.01, \"memo\":\"test_memo\"},\"sr1x7gcqdy670l2v4p9h2m4n5zgzde9y6ht86egffa0qrq40c6z329yfgvu8vyf99tgvnq4hwshvfxxhfzuyvz8dr3lt32j70x8l34japg73ca4w6z9x7c7ryd2gnafg9eg3gpr90gtunraw\":{\"amount\":0.01, \"memo\":\"\"}}"

Send two amounts to two different spark addresses setting memo:
> curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "mintspark", "params": ["{\"sr1xtw3yd6v4ghgz873exv2r5nzfwryufxjzzz4xr48gl4jmh7fxml4568xr0nsdd7s4l5as2h50gakzjqrqpm7yrecne8ut8ylxzygj8klttsgm37tna4jk06acl2azph0dq4yxdqqgwa60\":{\"amount\":1},\"sr1x7gcqdy670l2v4p9h2m4n5zgzde9y6ht86egffa0qrq40c6z329yfgvu8vyf99tgvnq4hwshvfxxhfzuyvz8dr3lt32j70x8l34japg73ca4w6z9x7c7ryd2gnafg9eg3gpr90gtunraw\":{\"amount\":0.01, \"memo\":\"test_memo2\"}}"] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/
```
curl example should be probably be update to:

`> curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "mintspark", "params": ["{"sr1xtw3yd6v4ghgz873exv2r5nzfwryufxjzzz4xr48gl4jmh7fxml4568xr0nsdd7s4l5as2h50gakzjqrqpm7yrecne8ut8ylxzygj8klttsgm37tna4jk06acl2azph0dq4yxdqqgwa60":{"amount":1},"sr1x7gcqdy670l2v4p9h2m4n5zgzde9y6ht86egffa0qrq40c6z329yfgvu8vyf99tgvnq4hwshvfxxhfzuyvz8dr3lt32j70x8l34japg73ca4w6z9x7c7ryd2gnafg9eg3gpr90gtunraw":{"amount":0.01, "memo":"test_memo2"}}"] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/`

example run - mintspark
![image](https://github.com/firoorg/firo/assets/48771525/e392e742-6285-41a1-bb9a-9e7923c2987a)

### spendspark
see also #1308 

curl example should be probably be update from:
`curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "spendspark", "params": ["{\"TR1FW48J6ozpRu25U8giSDdTrdXXUYau7U\":{\"amount\":0.01, \"subtractFee\": false},\"TuzUyNtTznSNnT2rPXG6Mk7hHG8Svuuoci\":{\"amount\":0.01, \"subtractFee\": true}, \"sr1hk87wuh660mss6vnxjf0syt4p6r6ptew97de3dvz698tl7p5p3w7h4m4hcw74mxnqhtz70r7gyydcx6pmkfmnew9q4z0c0muga3sd83h786znjx74ccsjwm284aswppqf2jd0sssendlj\":{\"amount\":0.01, \"memo\":\"\", \"subtractFee\": false},\"sr1x7gcqdy670l2v4p9h2m4n5zgzde9y6ht86egffa0qrq40c6z329yfgvu8vyf99tgvnq4hwshvfxxhfzuyvz8dr3lt32j70x8l34japg73ca4w6z9x7c7ryd2gnafg9eg3gpr90gtunraw\":{\"amount\":0.01, \"memo\":\"test_memo\", \"subtractFee\": false}}"] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/`

change to:

`curl --user myusername --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "spendspark", "params": [{"TR1FW48J6ozpRu25U8giSDdTrdXXUYau7U":{"amount":0.01, "subtractFee": false},"TuzUyNtTznSNnT2rPXG6Mk7hHG8Svuuoci":{"amount":0.01, "subtractFee": true}, "sr1hk87wuh660mss6vnxjf0syt4p6r6ptew97de3dvz698tl7p5p3w7h4m4hcw74mxnqhtz70r7gyydcx6pmkfmnew9q4z0c0muga3sd83h786znjx74ccsjwm284aswppqf2jd0sssendlj":{"amount":0.01, "memo":"", "subtractFee": false},"sr1x7gcqdy670l2v4p9h2m4n5zgzde9y6ht86egffa0qrq40c6z329yfgvu8vyf99tgvnq4hwshvfxxhfzuyvz8dr3lt32j70x8l34japg73ca4w6z9x7c7ryd2gnafg9eg3gpr90gtunraw":{"amount":0.01, "memo":"test_memo", "subtractFee": false}}] }' -H 'content-type: text/plain;' http://127.0.0.1:8332/`

example run - spendspark
![image](https://github.com/firoorg/firo/assets/48771525/c06dba36-b87d-4908-88e6-fd5f06d5e01b)

### Local setup
Ubuntu 18.04
![image](https://github.com/firoorg/firo/assets/48771525/18d8542f-3ffd-4966-baae-be5b86ac5d66)
![image](https://github.com/firoorg/firo/assets/48771525/f6555d2b-9827-4e59-bc91-c9bc7c34c2a9)

**Comment by mobias17** [comment]
`automintspark `and `antominlelantus `require one additional row with name of function in the documentation string to display correctly in the `help ` funtion

![image](https://github.com/firoorg/firo/assets/48771525/54400d64-49d8-4bd0-bb6b-6860c7ca240c)


![image](https://github.com/firoorg/firo/assets/48771525/e8d771f6-9e54-4874-a69f-fc8a9c480479)

**Comment by levonpetrosyan93** [comment]
Fixed with [881ff260d3b459d8b2e6eaa5c089eec35c385d47](https://github.com/firoorg/firo/commit/881ff260d3b459d8b2e6eaa5c089eec35c385d47) , Feel free to reopen, in case issue again happens.
